src/sflow/hsflowd/patch/0005-Use-close-range-syscall-over-blindly-looping-over-al.patch

```diff
@@ -0,0 +1,43 @@
+From dfc83d5cf6cbfa9056c4892d5ad0c3d3ac667d4f Mon Sep 17 00:00:00 2001
+From: vkarri <vkarri@contoso.com>
+Date: Tue, 21 Oct 2025 01:23:08 +0000
+Subject: [PATCH]  Use close range syscall over blindly looping over all FD
+
+getdtablesize is deprecated in Kernel 6.12 and this is returning a really large number.
```

Comment:
This _might be_ because of changes in Docker using a larger default value for the number of file descriptors allowed in a container, rather than any kernel change.